RadarSDK/RadarAPIClient.m

```diff
@@ -90,22 +105,16 @@ - (void)getConfig {
                                 sleep:NO
                            logPayload:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
-                        if (!res) {
-                            return;
-                        }
-        
-                        [Radar flushLogs];
+        [Radar flushLogs];
 
-                        id metaObj = res[@"meta"];
-                        if (metaObj && [metaObj isKindOfClass:[NSDictionary class]]) {
-                            NSDictionary *meta = (NSDictionary *)metaObj;
-                            id configObj = meta[@"config"];
-                            if (configObj && [configObj isKindOfClass:[NSDictionary class]]) {
-                                NSDictionary *config = (NSDictionary *)configObj;
-                                [RadarSettings setConfig:config];
-                            }
-                        }
-                    }];
+        if (!res) {
```

Comment:
out of curiosity, how come we're calling `flushLogs` - beforehand it seemed like we were short-circuiting `if (!res)`

Reply:
Good catch--I'll change it.